README.md

```diff
@@ -17,4 +18,24 @@ Confirm that the following requests work for you
 4. `curl http://localhost:3000/hello`
     * response: ```{"message": "hello world"}```
 
+## Local Database Connection
+
+**Guide that was used:** https://betterprogramming.pub/how-to-deploy-a-local-serverless-application-with-aws-sam-b7b314c3048c
+
+Follow these steps to get Dynamodb running locally
+
+1. **Create a Docker bridge network**
```

Comment:
nit: as a future commit/PR, should we include a default script that executes all these steps?

Reply:
Yes! I created a trello ticket for it https://trello.com/c/54YjlIKW/53-cfm47-script-to-setup-dev-database-environment